src/openrct2/network/NetworkBase.cpp

```diff
@@ -45,7 +45,7 @@ using namespace OpenRCT2;
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-#define NETWORK_STREAM_VERSION "4"
+#define NETWORK_STREAM_VERSION "5"
```

Comment:
This doesn’t change the network protocol, so this is not necessary. Could you revert it?